src/jit/gentree.cpp

```diff
@@ -1740,6 +1740,35 @@ bool GenTreeCall::IsHelperCall(Compiler* compiler, unsigned helper) const
     return IsHelperCall(compiler->eeFindHelper(helper));
 }
 
+void GenTreeCall::ReplaceCallOperand(GenTree** operandUseEdge, GenTree* replacement)
+{
+    assert(operandUseEdge != nullptr);
+    assert(replacement != nullptr);
+    assert(TryGetUse(*operandUseEdge, &operandUseEdge));
+
+    GenTree* originalOperand = *operandUseEdge;
+    *operandUseEdge = replacement;
+
+    const bool isArgument = (replacement != gtControlExpr) && ((gtCallType != CT_INDIRECT) ||
+        ((replacement != gtCallCookie) && (replacement != gtCallAddr)));
```

Comment:
Thanks for the suggestions. I've split the line at the `&&` and things seem to be fine.